libexec/rc/rc.conf

```diff
@@ -565,7 +565,7 @@ moused_nondefault_enable="YES" # Treat non-default mice as enabled unless
 moused_enable="NO"	# Run the mouse daemon.
 moused_type="auto"	# See man page for rc.conf(5) for available settings.
 moused_port="/dev/psm0"	# Set to your mouse port.
-moused_flags=""		# Any additional flags to moused.
+moused_flags="-A 1.5 -T 4" # Any additional flags to moused.
```

Comment:
Probably not needed for TrueOS, but might be useful on ISO media before distributions have had a chance to install their customized settings. Up to you guys on this one.